TTS/vocoder/models/wavernn.py

```diff
@@ -322,7 +322,7 @@ def inference(self, mels, batched=None, target=None, overlap=None):
 
         with torch.no_grad():
             if isinstance(mels, np.ndarray):
-                mels = torch.FloatTensor(mels).type_as(mels)
```

Comment:
I wasn't sure what to do here, suggestion @erogol ? Or this is fine ?

Reply:
I guess I was trying to move the tensor to the same device as the model's input. But right now it looks stupid :)

Reply:
I guess we can use the same trick we used above.